ui/src/components/shared/CredentialRevokeModal.tsx

```diff
@@ -18,22 +20,28 @@ export function CredentialRevokeModal({
   onRevoke: () => void;
 }) {
   const env = useEnvContext();
+  const { identifier } = useIdentityContext();
   const { notifyChange } = useIssuerStateContext();
 
   const [messageAPI, messageContext] = message.useMessage();
 
   const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [, setSearchParams] = useSearchParams();
 
   const { revNonce: nonce } = credential;
 
   const handleRevokeCredential = () => {
     setIsLoading(true);
 
-    void revokeCredential({ env, nonce }).then((response) => {
+    void revokeCredential({ env, identifier, nonce }).then((response) => {
       if (response.success) {
         onClose();
         onRevoke();
-
+        setSearchParams((previousParams) => {
+          const params = new URLSearchParams(previousParams);
+          params.set(REVOKED_SEARCH_PARAM, "true");
```

Comment:
Another leftover?

Please make sure you remove everything about this, including `REVOKED_SEARCH_PARAM`.